pilot/PostsViewController.swift

```diff
@@ -55,14 +61,17 @@ class HistoryViewController: UIViewController {
     self.navigationController?.navigationBar.isTranslucent = false
     self.navigationController?.navigationBar.shadowImage = UIImage()
 
-    self.navigationController?.navigationBar.topItem?.title = "History"
+    self.navigationController?.navigationBar.topItem?.title = "Posts"
     self.navigationController?.navigationBar.prefersLargeTitles = true
     self.navigationController?.navigationItem.largeTitleDisplayMode = .never
 
-    self.historyTableView.delegate = self
-    self.historyTableView.dataSource = self
+    self.postsTableView.delegate = self
+    self.postsTableView.dataSource = self
 
-    self.historyTableView.register(HistoryTableViewCell.self, forCellReuseIdentifier: "HistoryTableViewCell")
+    self.postsTableView.rowHeight = 50
```

Comment:
Making this non-constant will come later, right?

Reply:
Yea, that'll come with size classes later on